reputation/reputation.tex

```diff
@@ -36,26 +36,16 @@ \subsection{Earning and losing reputation}\label{sec:earning-losing-rep}
 \subsubsection{Reputation change from contributing to a task}\label{sec:earning-rep-from-task}
 Each task requires three roles to be assigned: the manager, the worker and the evaluator (as described in Section \ref{sec:tasks}). If the bounty for the task is denominated in the colony's token, each of these roles are eligible to earn reputation when the task is completed as long as their work was well received.
 
-The performance of the user who has completed the work is established when the work is submitted and then evaluated. At this point, the evaluator grades the work submitted by the worker, and the worker rates the manager's ability to coordinate delivery of the task\footnote{These scores should be submitted using a pre-commit and reveal scheme to ensure secrecy during the rating process and avoid retaliatory grading in the event that the manager and evaluator are the same person, which we expect to be a reasonably common occurrence. In the event of a user not committing or revealing within a reasonable time, their rating of their counterpart is assumed to be the highest possible and they receive a mildly negative rating. } out of five stars.
+The performance of the user who has completed the work is established when the work is submitted and then evaluated. At this point, the evaluator grades the work submitted by the worker, and the worker rates the manager's ability to coordinate delivery of the task\footnote{These scores should be submitted using a pre-commit and reveal scheme to ensure secrecy during the rating process and avoid retaliatory grading in the event that the manager and evaluator are the same person, which we expect to be a reasonably common occurrence. In the event of a user not committing or revealing within a reasonable time, their rating of their counterpart is assumed to be the highest possible and they receive a mild reputation penalty. } on a scale of one to three points.
 
-In the case of the evaluator, a rating of 0-2 stars counts as them rejecting the work, and a score of 3-5 stars counts as accepting the work. Beyond that, we suggest the following guidelines for ratings:
+In the case of the evaluator, a rating of 1 point counts as them rejecting the work and a score of 2-3 points counts as accepting the work. Therefore ratings and the reputation lost or gained follow these rules:
```

Comment:
> Therefore ratings and the reputation lost or gained follow these rules

The 'therefore' here implies that this is a logical consequence of the previous statement, which isn't really true. Can we change this to:

> The rating received defines the reputation change the user will experience: